com.playeveryware.eos/Runtime/Core/AuthenticationListener.cs

```diff
@@ -34,6 +34,15 @@ namespace PlayEveryWare.EpicOnlineServices
     /// </summary>
     public class AuthenticationListener: IAuthInterfaceEventListener, IConnectInterfaceEventListener, IDisposable
     {
+        /// <summary>
+        /// Identifies the kind of authentication change.
+        /// </summary>
+        public enum AuthenticationLevelChangeType
+        {
+            Auth,
+            Connect
+        }
```

Comment:
I think that a more appropriate name for this `enum` would be `AuthenticationChangeOrigin` to indicate from which system the authentication change originated.

Reply:
This was renamed to "LoginChangeKind". How does that feel?